lib/mix/tasks/ecto.drop.ex

```diff
@@ -19,6 +19,8 @@ defmodule Mix.Tasks.Ecto.Drop do
 
   """
 
+  @recursive true
```

Comment:
I wouldn't add this one just because it could end-up being very, very, very dangerous.